charts/istio-authorizer/values.yaml

```diff
@@ -62,35 +65,55 @@ serviceAccount:
 ##
 issuerURL: "https://acp.acp-system:8443/default/system"
 
+## Pod resources definition
+resources: {}
+  # requests:
+  #   cpu: 10m
+  #   memory: 48Mi
+  # limits:
+  #   cpu: 100m
+  #   memory: 96Mi
+
+## Define EnvoyFilter
+## Detailed configuration is available here
+## https://istio.io/latest/docs/reference/config/networking/envoy-filter/
+envoyFilter:
```

Comment:
I think we should rename this.

envoyFilter is technically what it is, but it's not clear what's the purpose.
The main goal is to add body parsing so I would use


```
## Enable body parsing
## If you want to use request body in your authorization policies, enable this flag.
## It's going to add a filter to Envoy which will propagate the request body to the authorizer. 
parseBody:
  enabled: false
```

Reply:
fixed